packages/tools/pie-css/scss/mixins/ _visuallyHidden.scss

```diff
@@ -0,0 +1,10 @@
+// Mixin to hide an element yet keep it available for screen readers.
```

Comment:
I have no idea if this is just github rendering it weirdly, but above it looks like this filename has a space at the start of it? (`mixins/ _visuallyHidden.scss`)

Reply:
good spot. it's an extra space